package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "nashviva",
+  "version": "1.0.0",
+  "description": "A digital pocketknife for life in Nashville",
+  "scripts": {
+    "lint": "standard --verbose | snazzy",
+    "fixcode": "standard --fix",
+    "precommit": "if [ -d 'ignite-base' ]; then cd ignite-base; fi; npm run lint"
```

Comment:
Whoops. I learned about this setup from the Ignite react-native starter. I think I have just been carrying that extra code in the project I copied it from. Removed.